data/tests/test_emission_calculations.py

```diff
@@ -220,7 +226,7 @@ def test_calculate_municipality_budgets(self):
         df_expected["budgetShare"] = [0.12539888902021, 0.87460111097979]
         df_expected["Budget"] = [10031911.1216168, 69968088.8783832]
 
-        df_result = calculate_municipality_budgets(df_input, LAST_YEAR_WITH_SMHI_DATA, CURRENT_YEAR, BUDGET_YEAR)
+        df_result = calculate_municipality_budgets(df_input, LAST_YEAR_WITH_SMHI_DATA, CURRENT_YEAR, 80000000, BUDGET_YEAR)
```

Comment:
Consistent total budget with other places where we use this value.

Might be worth clarifying what this number means in the test though. Perhaps assigning it to a named constant in the test file, which would describe its purpose.